# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.cpp
# Commit: 72d4bf5b093e
# Full Hash: 72d4bf5b093e9b00eed78bf4e51850d56df5ae6c
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 99b2e5c55a25 -r 72d4bf5b093e gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.cpp
--- a/gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.cpp	Thu Nov 05 09:14:52 2020 +0000
+++ b/gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.cpp	Thu Nov 05 09:15:05 2020 +0000
@@ -145,7 +145,8 @@
   return mSurface->GetYUVColorSpace();
 }
 
-bool RenderMacIOSurfaceTextureHost::MapPlane(uint8_t aChannelIndex,
+bool RenderMacIOSurfaceTextureHost::MapPlane(RenderCompositor* aCompositor,
+                                             uint8_t aChannelIndex,
                                              PlaneInfo& aPlaneInfo) {
   if (!aChannelIndex) {
     mSurface->Lock();